### PR TITLE
Pulling News from Django

### LINKED ISSUE #231: Connect News to Django data

### PR DESCRIPTION
**Pull request by gvn**
Closes #231

### CODE REVIEW

**Comment by gvn** [comment]
@gideonthomas Is there some reason dates aren't coming back from Django in ISO format? I'm seeing dates like: `2017-12-13` in News JSON.

File: scripts/fetch-json.js [hunk 1]
```diff
@@ -6,24 +6,18 @@ let chalk = require(`chalk`);
...
-request(`https://network-pulse-api-production.herokuapp.com/entries/?ordering=-created&page_size=996&format=json&featured=True&page=1`, (error, response, body) => {
-  if (!error && response.statusCode === 200) {
-    console.log(chalk.green(`Pulled Pulse JSON`));
-    shell.ShellString(body).to(`source/json/temp/pulse.json`);
-  } else {
-    console.log(chalk.red(`Failed to pull Pulse JSON!`));
-    shell.exit(1);
-  }
-});
+let fetch = (shortName, source) => {
```
Comment (alanmoo) [inline scripts/fetch-json.js]: Can you name this something else so it isn't confusing in the future? (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)

Comment (gvn) [inline scripts/fetch-json.js]: Do you mean the function name?

**Comment by gvn** [comment]
@gideonthomas – I had to add another field to the schema: `excerpt`, which is just a string.

**Comment by gvn** [comment]
Related / Follow-on: #236

**Pull request merged**